### PR TITLE
make puli requirement more flexible

### PR DESCRIPTION
**Pull request by dbu**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -11,9 +11,9 @@
         }
     ],
     "require": {
-        "php": ">=5.4",
-        "puli/composer-plugin": "1.0.0-beta8",
-        "puli/discovery": "1.0.0-beta8"
+        "php": "^5.4|7.*",
```
Comment (sagikazarmark) [inline composer.json]: Is this better than ^7.0?

Comment (dbu) [inline composer.json]: its semantically the same. maybe caret is better for when we need to bump
to some php 7 that includes a fix (if we run into that situation). but we
can also change once it becomes relevant

Comment (sagikazarmark) [inline composer.json]: I don't mind if it's the same.

**Comment by sagikazarmark** [comment]
:+1:

**Pull request merged**